src/SFA.DAS.EmployerFinance/Commands/RefreshEmployerLevyData/LevyImportCleanupStrategy.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.EmployerFinance.Data;
+using SFA.DAS.EmployerFinance.Extensions;
+using SFA.DAS.EmployerFinance.Models.Levy;
+using SFA.DAS.EmployerFinance.Services;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.EmployerFinance.Commands.RefreshEmployerLevyData
+{
+    public class LevyImportCleanerStrategy : ILevyImportCleanerStrategy
+    {
+        private readonly IDasLevyRepository _dasLevyRepository;
+        private readonly IHmrcDateService _hmrcDateService;
+        private readonly ILog _logger;
+        public LevyImportCleanerStrategy(
+            IDasLevyRepository dasLevyRepository,
+            IHmrcDateService hmrcDateService,
+            ILog logger)
+        {
+            _dasLevyRepository = dasLevyRepository;
+            _hmrcDateService = hmrcDateService;
+            _logger = logger;
+        }
+
+        public async Task<IEnumerable<DasDeclaration>> Cleanup(string empRef, IEnumerable<DasDeclaration> declarations)
+        {
+            var temp = declarations.OrderBy(c => c.SubmissionDate).ToArray();
```

Comment:
Rename temp to something I won't forget in 4 lines. :)

Reply:
Renamed